src/components/visual-dropdown/styles/editor.scss

```diff
@@ -186,29 +169,13 @@
 	width: 63px;
 }
 
-.components-coblocks-visual-dropdown__popover {
```

Comment:
Let's make sure this doesn't affect the Map block's styles dropdown (if you have a Google Map API Key)

Reply:
It was indeed effected. I had to adjust lines 175-181 to get the styles to look correct but that is now adjusted. Looks right in 5.2.4 and 5.3. Looks good with and without GB running.